packages/dropdowns/src/Menu/Menu.js

```diff
@@ -37,6 +37,18 @@ const Menu = props => {
     popperReferenceElementRef,
     downshift: { isOpen, getMenuProps }
   } = useDropdownContext();
+  const scheduleUpdateRef = useRef(undefined);
+
+  useEffect(() => {
+    /**
+     * Recalculate popper placement while open to allow animations to complete.
+     * This must be ran every render to allow for the number of items to change
+     * and still be placed correctly.
+     **/
+    if (isOpen) {
+      scheduleUpdateRef.current && scheduleUpdateRef.current();
+    }
```

Comment:
Either:

```js
if (isOpen && scheduleUpdateRef.current) {
  scheduleUpdateRef.current();
}
```

or 

```js
isOpen && scheduleUpdateRef.current && scheduleUpdateRef.current();
```

Reply:
Mhh thinking about it I think we should keep it the way it is, that will allow more code to be added to this if statement later.